lib/_http_outgoing.js

```diff
@@ -756,6 +765,12 @@ function write_(msg, chunk, encoding, callback, fromEnd) {
     return false;
   }
 
```

Comment:
What if content length actually is 0?

Reply:
Never thought about it, we could check in storeHeader() if the content-length is bigger than 0 in this case, but I think it would be great to establish a minimum size.

Reply:
A new error will also be needed, what if the content-length is also a negative value... either the 0 or negative values should be errors, it could be ERR_INVALID_CONTENT_LENGTH

Reply:
i think 0 is valid

Reply:
Ok I did some tests, and when the Content-Length is 0 it just passes the test, but when it's a negative value it should throw a new error, I think the best to do is verify if the value from the header is positive.